docker-compose.local.yml

```diff
@@ -46,6 +46,11 @@ services:
       - URI_QUESTION_SERVICE=http://question-service:8500
       - ACCESS_TOKEN_SECRET=7c0dd92cafc2537595f48eca543a45e38e25b0322c558b8cb44907db49726b54a0809709c4491cbc086187009367ac4febfb8d5db310320c16d510818d833ad9
 
+  peerjs-server:
+    image: peerjs/peerjs-server
+    ports:
+      - "9000:9000"
+
   postgres_db:
     image: postgres:latest
     restart: always
```

Comment:
As discussed on Monday, perhaps can help to remove unnecessary environment variables on line 94/95 pls
>       WATCHPACK_POLLING: "true"
>      CHOKIDAR_USEPOLLING: "true"

Reply:
Not meant to be commented out, keep accidentally committing this, sorry